tests/baselines/reference/tscWatch/consoleClearing/when-preserveWatchOutput-is-true-in-config-file/createWatchOfConfigFile.js

```diff
@@ -43,6 +43,10 @@ Shape signatures in builder refreshed for::
 /f.ts (used version)
 /a/lib/lib.d.ts (used version)
 
+PolledWatches::
+/node_modules/@types: *new*
+  {"pollingInterval":500}
+

```

Comment:
Can you explain why these were missing before and are added now? I don’t quite see why from the implementation change.

Reply:
Sorry i thought this was also handled by "canWatchFileOrDirectory" thing but we had logic for watching "node_modules/@types" at tsconfig or jsconfig directory as well which has some issues when it is "/". I will look more into this and fix any other issues with config at root directory.